### ISSUE TITLE
fm:player:add event doesn't contain uuid

### ISSUE THREAD

**Issue by edoparearyee** [thisissoon/FM-API#90]
the uuid is needed to be able to delete tracks without having to query the queue endpoint:

Example of what's currently received:

```json
{ 
  "event": "add",
  "uri": "spotify:track:7hDc8b7IXETo14hHIHdnhd",
  "user": "ec08fc20-c02c-4c86-9c4d-84e5aed5e5a1"
}
```

**Comment by krak3n** [comment]
I should get to this on Friday

**Comment by krak3n** [comment]
Fixed in c7b7eab - it's called `id` rather than `uuid`

**Comment by krak3n** [comment]
deployed
